man/Makefile.am

```
@@ -1,72 +1,93 @@
 
-CLEANFILES			= *~
+CLEANFILES			= *~ $(generated_in_man) $(generated_md_man)
+DISTCLEANFILES			= $(CLEANFILES)
 MAINTAINERCLEANFILES		= Makefile.in
-DISTCLEANFILES			= $(generated)
-BUILT_SOURCES			= $(generated)
-EXTRA_DIST			= $(templates) $(allstatic)
 
-man_MANS			= \
-	$(generated)		\
-	$(allstatic)
+BUILT_SOURCES			=
+EXTRA_DIST			= $(generated_in_man) $(generated_md_man) \
+				  $(templates_md_in)  $(templates_md_src) \
+				  $(templates_in)
 
-allstatic			= \
-	routes.5
+if pandoc
+DISTCLEANFILES			+= $(templates_md_in)
+MAINTAINERCLEANFILES		+= $(templates_md_in)
+BUILT_SOURCES			+= $(templates_md_in)
+endif
 
-generated			= \
-	wicked-config.5		\
-	wicked.7		\
-	wicked.8		\
-	wickedd.8		\
-	wicked-ethtool.8	\
-	wicked-firmware.8	\
-	wicked-redfish.8	\
+man_MANS			= \
+	$(generated_md_man)	\
+	$(generated_in_man)
+
+generated_in_man		= \
+	ifcfg-bond.5		\
 	ifcfg-bonding.5		\
 	ifcfg-bridge.5		\
-	ifcfg-dummy.5		\
-	ifcfg-ipoib.5		\
+	ifcfg-dhcp.5		\
 	ifcfg-infiniband.5	\
+	ifcfg-lo.5		\
 	ifcfg-macvlan.5		\
 	ifcfg-macvtap.5		\
 	ifcfg-ovs-bridge.5	\
 	ifcfg-ppp.5		\
-	ifcfg-team.5		\
 	ifcfg-tunnel.5		\
 	ifcfg-vlan.5		\
 	ifcfg-vxlan.5		\
-	ifcfg-wireless.5	\
-	ifcfg-dhcp.5		\
-	ifcfg-lo.5		\
 	ifcfg.5			\
+	ifdown.8		\
 	ifroute.5		\
 	ifrule.5		\
-	ifsysctl.5		\
-	ifdown.8		\
 	ifstatus.8		\
-	ifup.8
+	ifsysctl.5		\
+	ifup.8			\
+	routes.5		\
+	wicked-config.5		\
+	wicked-ethtool.8	\
+	wicked.7		\
+	wicked.8		\
+	wickedd.8
+
+generated_md_man		= \
+	ifcfg-dummy.5		\
+	ifcfg-ipoib.5		\
+	ifcfg-team.5		\
+	ifcfg-wireless.5	\
+	wicked-firmware.8	\
+	wicked-redfish.8
+
 
+templates_md_src		= $(addprefix src/,$(generated_md_man:=.md))
+templates_md_in			= $(generated_md_man:=.in)
+templates_in			= $(generated_in_man:=.in)
 
-templates			= $(generated:=.in)
+# pandoc would write @VAR@ as [at]VAR[at] to .in:
+#   .md sources are using %VAR% variables, while
+#   .in templates are using common autoconf @VAR@
+substitute_md_vars		= \
+	$(AM_V_GEN)sed		-e "s|[%]\(.*\)[%]|@\\1@|g"
```

The replacement is to bold. Take a look to https://github.com/openSUSE/wicked/blob/3585f97e533dcb7ee80d5096b145e1500b9d476e/man/src/ifcfg-wireless.5.md?plain=1#L242 which is in trouble with this regex.

So something like:
```suggestion
	$(AM_V_GEN)sed -E 's/%(wicked_(piddir|sbindir|statedir|storedir|schemadir|configdir|supplicantdir|extensionsdir)|PACKAGE_BUGREPORT)%/@\1@/g'
```
or
```suggestion
	$(AM_V_GEN)sed -E			\
	-e "s|%(wicked_piddir)%|@\1@|g"		\
	-e "s|%(wicked_sbindir)%|@\1@|g"	\
	-e "s|%(wicked_statedir)%|@\1@|g"	\
	-e "s|%(wicked_storedir)%|@\1@|g"	\
	-e "s|%(wicked_schemadir)%|@\1@|g"	\
	-e "s|%(wicked_configdir)%|@\1@|g"	\
	-e "s|%(wicked_supplicantdir)%|@\1@|g"	\
	-e "s|%(wicked_extensionsdir)%|@\1@|g"	\
	-e "s|%(PACKAGE_BUGREPORT)%|@\1@|g"
```